galicaster/classui/recorderui.py

```diff
@@ -59,10 +59,10 @@
 # No-op function for i18n
 def N_(string): return string
 
-STATUS = [  [N_("Initialization"),"#F7F6F6"],
-            [N_("Waiting"),"#F7F6F6"],
+STATUS = [  [N_("Initialization"),"#484848"],
+            [N_("Waiting"),"#484848"],
             [N_("Recording"),"#FF0000"],
-            [N_("Paused"),"#F7F6F6"],
+            [N_("Paused"),"#484848"],
             [N_("Error"),"#FF0000"],
             ]
 
```

Comment:
TODO: Write in the global variables the background and the foreground, not only one of them.